skrub/_dataframe/_test_utils.py

```diff
@@ -0,0 +1,6 @@
+def is_namespace_pandas(px):
```

Comment:
I think "module" would be more precise than "namespace"

Reply:
I agree, done